superduperdb/base/datalayer.py

```diff
@@ -117,8 +117,13 @@ def server_mode(self, is_server: bool):
         self._server_mode = is_server
 
     def initialize_vector_searcher(
-        self, identifier, searcher_type: t.Optional[str] = None, backfill=False
+        self, identifier, searcher_type: t.Optional[str] = None
     ) -> t.Optional[BaseVectorSearcher]:
+        '''
```

Comment:
Please use `"""` (this is standard style).

Reply:
Used for all docstrings in components

Reply:
Yes here also please (everywhere).